src/app/components/tree-view/index.html

```diff
@@ -0,0 +1,209 @@
+<sky-tabbed-demo-page
+  pageTitle="Tree view"
+  navTitle="Tree view"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      Tree views display a hierarchical view of items and have multiple modes for selecting items.
+    </p>
+    <sky-alert alertType="warning">
+      This component is a release candidate. We do not plan to make any breaking changes the component before the official release, but such changes are possible.
+    </sky-alert>
+  </sky-demo-page-summary>
+
+  <div class="sky-margin-stacked-separate">
+    <sky-demo-page-example>
+      <sky-angular-tree-view-demo></sky-angular-tree-view-demo>
+      <sky-demo-page-code demoName="Angular tree view"></sky-demo-page-code>
+    </sky-demo-page-example>
+  </div>
+
+<sky-tabset id="content-tabs">
+
+  <sky-tab
+    tabHeading="Design">
+
+    <sky-demo-page-content sectionHeading="Usage">
+      <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+              <p>Use tree views to show lists of hierarchical items. They can be used in view-only or selection contexts.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/angular-tree-view/use-when.png" />
+              <div class="caption caption-success sky-text-success">Do use tree views to show lists of hierarchical items.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Anatomy">
+      <sky-docs-anatomy imagePath="~/assets/img/guidelines/angular-tree-view/anatomy.png">
+        <sky-docs-anatomy-item anatomyNumber="1">Expand/collapse button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">List item content</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Checkbox</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Context menu</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Options">
+      <h3>Selection</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>You can use single- or multi-select mode when you use a tree view in a selection context.</p>
```

Comment:
As written, the hyphen is correct. (If you remove it, `single` is no modifying the `select` in `multi-select`, so you are effectively saying `single mode` instead of `single-select-mode`.)

However, we do not hyphenate `multiselect` elsewhere in the SKY UX docs, so we shouldn't hyphenate it here. And if we delete the hyphen in that word, then the hyphen after `single` becomes stray punctuation with no meaning. So we should change `multi-select` to `multiselect` and also change `single-` to `single-select`.